docs/Python-API.md

```diff
@@ -132,6 +124,79 @@ variable named `env` in this example, can be used in the following way:
   Sends a shutdown signal to the environment and closes the communication
   socket.
 
+### Modifying the environment from Python
+The Environment can be modified by using side channels to send data to the
+environment. When creating the environment, pass a list of side channels as
+`side_channels` argument to the constructor.
+
+__Note__ : A side channel will only send/receive messages when `env.step` is
+called.
+
+#### EngineConfigurationChannel
+An `EngineConfigurationChannel` will allow you to modify how
+fast and how accurate the Unity engine is. For example, modifying the
```

Comment:
nit: Not sure accurate is the correct word here.  Maybe

An `EngineConfiguration` will allow you to modify the time scale and graphics quality of the Unity engine.

Perhaps the second sentence is redundant.